### PR TITLE
Add README for storytelling resources

### PR DESCRIPTION
**Pull request by fsmunoz**
Improve repo navigation by adding a README to the storytelling directory, with a description of what's covered in individual documents.

### CODE REVIEW

File: communication/contributor-comms/storytelling-resources/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Storytelling resources
```
Comment (chris-short) [inline communication/contributor-comms/storytelling-resources/README.md:1]: Please update to Blogging Resources

File: communication/contributor-comms/storytelling-resources/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+We have collected some information to get anyone started:
```
Comment (chris-short) [inline communication/contributor-comms/storytelling-resources/README.md:3]: Add a little bit to this affect: "If you're looking to get started blogging on the K8s.dev blog this page contains a list of resources available to help you be successful."

File: communication/contributor-comms/storytelling-resources/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## [Multichannel communications](./multichannel-communications.md)
+
+A guide to facilitate identification of the correct communication
+channel, both for interacting with the Kubernetes community, and also for outbound communication.
+
+## [Social media guidelines](./social-guidelines.md)
+
+Guidelines for social media, currently focused on the
+[K8sContributor](https://twitter.com/k8scontributors) account, managed
+by us.
```
Comment (chris-short) [inline communication/contributor-comms/storytelling-resources/README.md:35]: Please add Mastodon: https://hachyderm.io/@K8sContributors

**Review by chris-short — CHANGES_REQUESTED** [review]
Good stuff

**Comment by fsmunoz** [comment]
@chris-short updated with the suggestions.

File: communication/contributor-comms/storytelling-resources/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## [Multichannel communications](./multichannel-communications.md)
+
+A guide to facilitate identification of the correct communication
+channel, both for interacting with the Kubernetes community, and also
+for outbound communication.
+
+## [Social media guidelines](./social-guidelines.md)
+
+Guidelines for social media, currently focused on the
+[K8sContributor](https://twitter.com/k8scontributors) Twitter account
```
Comment (AvineshTripathi) [inline communication/contributor-comms/storytelling-resources/README.md:35]:
```suggestion
[@K8sContributors](https://twitter.com/k8scontributors) Twitter account
```

**Comment by kaslin** [comment]
Let's move the folder to something like "blogging resources" while we're at it.

**Comment by fsmunoz** [comment]
@chris-short @kaslin , done in the latest commit.

File: communication/contributor-comms/storytelling-resources/README.md (hunk 4 above)
Comment (fsmunoz) [reply communication/contributor-comms/storytelling-resources/README.md]: Good one, changed in the last commit (this was outdated due to the directory rename).

**Comment by fsmunoz** [comment]
Rebased just in case there are no further changes.

**Comment by AvineshTripathi** [comment]
lgtm

**Pull request merged**